addons/payment/models/payment_acquirer.py

```diff
@@ -198,6 +198,18 @@ def _ensure_payment_method_line(self, allow_create=True):
             pay_method_line.journal_id = self.journal_id
         elif allow_create:
             default_payment_method_id = self._get_default_payment_method_id()
+
+            pay_method_line = self.env['account.payment.method.line'].search(
```

Comment:
IMO this search is the one that should replace line 187.
What's the point of searching 2 times?